code/iaas/accounts/src/main/resources/META-INF/cattle/accounts/spring-accounts-context.xml

```diff
@@ -149,10 +153,29 @@
             </list>
         </property>
     </bean>
-    
+
     <bean id="Project" class="io.cattle.platform.api.schema.ObjectBasedSubSchemaFactory" >
         <property name="id" value="project" />
-        <property name="schemaFactory" ref="User" />
+        <property name="schemaFactory" ref="AdminSchema" />
+        <property name="postProcessors">
+            <list>
+                <bean class="io.cattle.platform.schema.processor.JsonFileOverlayPostProcessor" >
+                    <property name="path" value="schema/project" />
+                </bean>
+                <bean class="io.cattle.platform.schema.processor.AuthOverlayPostProcessor" >
+                    <property name="resources" >
+                        <bean class="io.cattle.platform.spring.resource.SpringUrlListFactory" >
+                            <property name="resources">
+                                <list>
+                                    <value>classpath:schema/user/user-auth.json</value>
```

Comment:
Right now this file is different from what it will be because until i do a big refactor of the cattle integration and validation tests you can't have the correct permissions. The end result after the follow up pr will be that a user account will not be able to do anything except create projects and projectMembers as well as api Keys and project accounts will be able to do what a user used to be able to do. This way all resources have to be created and managed by accounts that are a project or agent etc. But not a user.